sql/hive-thriftserver/src/test/scala/org/apache/spark/sql/hive/thriftserver/ThriftServerWithSparkContextSuite.scala

```diff
@@ -125,9 +125,23 @@ trait ThriftServerWithSparkContextSuite extends SharedThriftServer {
     withCLIServiceClient(clientUser) { client =>
       val sessionHandle = client.openSession(clientUser, "")
       val confOverlay = new java.util.HashMap[java.lang.String, java.lang.String]
-      val opHandle = client.executeStatement(sessionHandle, sql, confOverlay)
-      val rowSet = client.fetchResults(opHandle)
-      assert(rowSet.toTRowSet.getColumns.get(0).getStringVal.getValues.get(0) === clientUser)
+      val exec: String => OperationHandle = client.executeStatement(sessionHandle, _, confOverlay)
+
+      exec(s"set ${SQLConf.ANSI_ENABLED.key}=false")
```

Comment:
can we test this in the sql/core side? IIRC you also added a test in sql/core.

Reply:
adressed